src/Illuminate/Support/ReadableHigherOrderCollectionProxy.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Illuminate\Support;
+
+class ReadableHigherOrderCollectionProxy
+{
+    /**
+     * The collection being operated on.
+     *
+     * @var \Illuminate\Support\Collection
+     */
+    protected $collection;
+
+    /**
+     * The method being proxied.
+     *
+     * @var string
+     */
+    protected $method;
+
+    /**
+     * Create a new proxy instance.
+     *
+     * @param  \Illuminate\Support\Collection  $collection
+     * @param  string  $method
+     * @return void
+     */
+    public function __construct(Collection $collection, $method)
+    {
+        $this->method = $method;
+        $this->collection = $collection;
+    }
+
+    public static function perform(Collection $collection, $method)
+    {
+        $action = 'filter';
```

Comment:
This has been defaulted to `filter` but it will never actually get to this line and not have a value that will match in the foreach below, because of the check made in Collection.php